.github/workflows/UpdateContentOverview.yml

```diff
@@ -6,6 +6,15 @@ on:
     paths:
       - packages/**/*
       - scripts/data/**/*
+  pull_request:
+    branches:
```

Comment:
Why should this run in every PR? Isnt it supposed to run only when we merge on master?

Reply:
well, yes, but I think the fact that it started failing on master shows that running it on PRs as well is a good idea. It would have avoided the problem I now created.

Reply:
But the problem you introduced was touching the scripts, imagine that for every widget being changed this pipeline will be triggered, even though is not needed.. I think is a waste of time to introduce it generally in all PRs as it does add value for features being developed within widgets.

Reply:
True. It's run duration is very short, but indeed, it doesn't add anything. What if limit it to just data/scripts?

Reply:
Thats a better idea :)

Reply:
done